pkg/controller/persistentvolume/persistentvolume_claim_binder_controller.go

```diff
@@ -33,6 +33,88 @@ import (
 	"github.com/golang/glog"
 )
 
+/*
+What states can a volume be in?  Can a volume be in more than one state at a time?
+
+------------------------------------------------------------------------
+
+case VolumePending:
```

Comment:
I think we can deprecate 'Available' entirely with the flow as described in the PNGs and the State objects in the prototype.